library/psa_crypto.c

```diff
@@ -1493,7 +1540,12 @@ psa_status_t psa_get_key_attributes( psa_key_handle_t handle,
 
     switch( slot->attr.type )
     {
-#if defined(MBEDTLS_RSA_C)
+#if defined(MBEDTLS_PSA_BUILTIN_ALG_RSA_PKCS1V15_CRYPT) || \
+    defined(MBEDTLS_PSA_BUILTIN_ALG_RSA_PKCS1V15_SIGN) || \
+    defined(MBEDTLS_PSA_BUILTIN_ALG_RSA_OAEP) || \
+    defined(MBEDTLS_PSA_BUILTIN_ALG_RSA_PSS) || \
```

Comment:
There's no driver interface to obtain the public exponent of an RSA key, so here there's no way to use a driver. However, it's a limitation we can live with for now (filed as https://github.com/ARMmbed/mbedtls/issues/3887). Returning the public exponent in the attributes is a minor feature that's only needed for API regularity (because `psa_generate_key` should support the public exponent attribute), and that feature is an Mbed TLS extension that is planned for addition to the official PSA API but maybe not in its current form. So this should be guarded by the RSA key pair types.